dozer/cogs/teams.py

```diff
@@ -61,6 +64,49 @@ async def teamsfor(self, ctx, user: discord.Member = None):
     `{prefix}teamsfor member` - Returns all team associations with the mentioned user. Assumes caller if blank.
     """
 
+    @command()
+    @guild_only()
+    async def compcheck(self, ctx, event_type: str, event_key):
+        """Allows you to see people in the Discord server that are going to a certain competition."""
+        if event_type.lower() == "frc":
+            teams_raw = await TBA(ctx.bot).session.event_teams(event_key)
+            teams = [team.team_number for team in teams_raw]
+        elif event_type.lower() == "ftc":
+            teams_raw = json.loads(await TOA(ctx.bot).parser.req(f"/api/event/{event_key}/teams"))
+            teams = [team['team']['team_number'] for team in teams_raw]
+        else:
+            raise BadArgument("Unknown event type!")
+        found_mems = False
+        e = discord.Embed(type='rich')
+        e.title = 'Members going to {}'.format(event_key)
+        for team in teams:
+            members = await TeamNumbers.get_by(team_type=event_type.lower(), team_number=team)
+            memstr = ""
+            for member in members:
+                mem = ctx.guild.get_member(member.user_id)
+                if mem is not None:
+                    newmemstr = "{} {} \n".format(mem.display_name, mem.mention)
+                    if len(newmemstr + memstr) > 1023:
+                        e.add_field(name=f"Team {team}", value=memstr)
+                        memstr = ""
+                    memstr += newmemstr
+                    found_mems = True
+            if len(memstr) > 0:
+                if len(e.fields) == 25:
+                    await ctx.send(embed=e)
+                    e = discord.Embed(type='rich')
+                    e.title = 'Members going to {}'.format(event_key)
+                e.add_field(name=f"Team {team}", value=memstr)
+
+        if not found_mems:
+            raise BadArgument("Couldn't find any team members for that event!")
+        else:
+            await ctx.send(embed=e)
+
+    teamsfor.example_usage = """
```

Comment:
I think you forgot to make the example usage. 

Also, given that very few FTC events even publicly publish team lists before events, it'll be a while before they even benefit from this one. I remember back in the day, a certain somebody would have a channel that would list all the weekly upcoming events and who was coming to each one - I wonder if we could do something similar with the bot?

Reply:
I did, pushing that now. No idea about the FTC one, but I'm going to get yelled at by someone if I leave it out. Not sure how I feel about the last thing